Doc/user_manual/six.tex

```diff
@@ -747,6 +747,62 @@ \subsubsection{Other element types}
 Some other elements, such as dipole edge (24), solenoid (25), multipole RF kicks ($\pm$26, $\pm$27, $\pm$28) are accepted by SixTrack, but they are not currently supported by the development team or tested for correctness.
 It is therefore advised to not use these elements.
 
+How to use the RF multipoles (from MadX to SixTrack):
+
+\subsubsection{1st order multipole (i.e. dipole)}
```

Comment:
The dipole is already documented above, in \subsubsection{Crab Cavity} - it would be good to merge these sections somehow...